README.md.handlebars

```diff
@@ -230,6 +232,12 @@ docker-compose.yaml, it will ignore an invalid GPG key during the installation.
 
 *Please note though*, that this affects the security of the system and should not be used constantly.
 
+### When i try to start cloud control, it keeps exiting with "./ccc: exit status 1" and i want to debug that more
+
+Use the `docker logs` command with the failed container to see the complete log output. You can enhance the log by using the
```

Comment:
I wonder if we need to give more info on how to get the name of the container? I use `docker ps -a` to fetch the id, but you can also use the container name, which is generated by docker compose (<current path>_cli_1).

Reply:
Okay.